blog/2020-12-18-How-is-office-culture-produced-maintained-transformed.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How is office culture produced, maintained, and transformed?
+author: Po Yee Ooi
+description: Writing our own library to parse and render Servant API types as documentation
+tags: culture
+---
+
+## Introduction
+
+An organisation's culture consists of the human interactions within it; the make-up of its people, the diversity of ideas, the friendliness of colleagues and the interactions between the office’s policies and the people who use them. How are these carried out in the best (and only) product team? How is the office culture here created, maintained and transformed? I interviewed Subra (Chief Technology Officer), Oishik (Software Engineer), Gracia (Associate Product Manager) and Jolanda (Software Engineer) to find out.
+
+## Act I: Making culture out of nothing at all
+
+### How do you create culture?*
+
+It is not straightforward, and certainly “not something you have control over”, says Subra, who has been with Holmusk since 2015. Workplace culture is organic, and a function of the type of people hired into the company. Just like the personalities of the employees and co-workers, the personality of the company is not something that can be changed easily. Thus, I think the key in creating a good workplace culture lies in the very first step: hiring the right people.
+
+The right person at Holmusk is both work-oriented and people-oriented, and neither quality is solely sufficient. Employees have to care about the work they do while having camaraderie with the rest of the team. The hiring team “wasn’t interested in the best programmers or people who could write an app in a day, but people who can deliver a product together as a team”, adds Oishik, who has worked with Holmusk since starting here as an intern in 2019. Having people who can work well together and be friends makes life easier and work more productive and enjoyable. This is especially true in the case of meetings, which often take up a significant proportion of one's working hours, making it a draining task. Having meetings with people you enjoy being around can therefore have a big positive impact on the enjoyability of the work overall.
+
+### How do you hire the right people?
+
+It appears that the key to finding out if someone is the right fit lies in this sifting tray called the team lunch. This team lunch, which introduces the new candidate to 4-5 other representatives from various teams (e.g. UX, Software, Product, etc.) over a casual meal, happens during the hiring process to understand if the applicant fits well with the rest of the existing team. “When a new person joins, it is important for us to meet them physically to about them and what they want to learn and experience,” says Gracia, who has worked with Holmusk since 2018.
```

Comment:
@jhrcek and @kwannoel , I think the word is "learn about them", but even then, the sentence is slightly awkward. Maybe it can be phrased as:
" When a new person joins, it is important for us to meet them physically to understand what they want to learn and experience"